static/js/containers/TermsOfServicePage.js

```diff
@@ -2,8 +2,13 @@
 import React from 'react';
 import Button from 'react-bootstrap/lib/Button';
 import { connect } from 'react-redux';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import UserImage from '../components/UserImage';
 
 import { saveProfile, FETCH_SUCCESS } from '../actions';
+import {
+  DASHBOARD_COURSE_HEIGHT,
+} from '../constants';
```

Comment:
I agree, the CSS classes used in the terms of service should be specific to the terms of service, or general to the entire dashboard piece of the site